src/spatial/camera.hpp

```diff
@@ -56,13 +59,36 @@ class Camera {
 
     // Accessor Methods to Camera Parameters
 
+   /**
+     * Returns the X resolution of this camera
+     * 
+     * @return the X resolution of this
+     */
     int XResolution() const { return xResolution; }
+
+    /**
+     * Returns the Y resolution of this camera
+     * 
+     * @return the Y resolution of this
+     */
     int YResolution() const { return yResolution; }
+
+    /**
+     * Returns the focal length of this camera
+     * 
+     * @return The focal length of this
+     */
     decimal FocalLength() const { return focalLength; }
+
     decimal Fov() const;
 
     // Mutator Method for Cameras
 
+    /**
+     * Sets the focal length of this
```

Comment:
Specify units, this has been a cause of confusion for LOST

Reply:
Answering by commenting the camera object to be in SI.